### DISCUSSION TITLE
Update Crazyflie and AIdeck firmware

### DISCUSSION THREAD

**Discussion by dhamidou** [bitcraze/discussions#1200] [Q&A] [answered]
Hi, I'm using crazyflie for development, but when I update the firmware of Crazyflie and AIdeck, the console does not show "ESP32: I (910) SYS: Initialized".
This is what the console shows me : 

![Capture d’écran de 2024-02-28 10-56-38](https://github.com/bitcraze/discussions/assets/95414130/82d00dab-f193-46d2-99e5-163e44f1192a)
Thank you

**Comment by gemenerik** [comment]
I presume you updated through cfclient? Could you retry flashing? Is LED1 flashing @2Hz?

**Reply by dhamidou** [reply]
LED1 flashing with 0.5 Hz

**Reply by dhamidou** [reply]
When i try to do this [https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/](url), this is what the terminal displays
![Capture d’écran de 2024-02-28 13-10-59](https://github.com/bitcraze/discussions/assets/95414130/666dd885-017e-4fbd-a8b4-a542c0e2f1b1)
![Capture d’écran de 2024-02-28 13-10-53](https://github.com/bitcraze/discussions/assets/95414130/35386bd3-a9cf-45a3-825b-158ab12a7c29)

**Reply by gemenerik** [reply]
Can you try upgrading your cfclient and cflib to the latest version if they are not already?
If you installed client and lib from latest release with pip, run `pip install --upgrade cfclient cflib`. Otherwise if you installed from source pull latest from git

**Reply by dhamidou** [reply]
I have the latest version

**Reply by dhamidou** [reply]
Every time I try to flash ESP or the bootloader it shows me : nRF51 has: sd-s130 and requires None and upgrade provides None. Current bootloader version is [2024.2.0] but upgrade provides [None]

**Reply by dhamidou** [reply, marked as answer]
ok, I solve the problem. If someone encounters this problem, you must update esp firmware by following this link : [https://github.com/bitcraze/aideck-esp-firmware](https://github.com/bitcraze/aideck-esp-firmware).

![Capture d’écran de 2024-02-29 10-08-42](https://github.com/bitcraze/discussions/assets/95414130/4af46fcb-6dc9-44e1-ba8e-100de7746500)
